stdlib/config-api/src/main/ballerina/src/config/config.bal

```diff
@@ -149,6 +149,14 @@ public function getAsMap(@untainted string key) returns map<any> {
     }
 }
 
+# Retrieves the specified configuration value as an array. If there is no mapping, an empty array will be returned.
+#
+# + key - The configuration to be retrieved
+# + return - Configuration value mapped by the key
+public function getAsArray(@untainted string key) returns any[] {
```

Comment:
On second thoughts, shall we make this `anydata[]` instead of `any[]`? I just tested this and with an `any[]`, you can't really do anything useful easily; you have to cast each of the array element to its required type. With `anydata[]`, you can do the following:
```ballerina
import ballerina/config;
import ballerina/io;

public function main() {
	int[]|error ports = int[].constructFrom(config:getAsArray("ports"));
	if (ports is int[]) {
		io:println(ports); 
	}
}
```

Reply:
Other than that, this is working as expected.

Reply:
Fixed this in 91dce9a970344c437ca6058e10f25ba7641179a2